backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/FunctionInlining.kt

```diff
@@ -144,8 +144,16 @@ private class Inliner(val globalSubstituteMap: MutableMap<DeclarationDescriptor,
     }
 
     //-------------------------------------------------------------------------//
-
-    private fun inlineFunction(callee: IrCall,                                              // Call to be substituted.
+    /**
+     * TODO: JVM inliner crushed on attempt inline this function from transform.kt
+     */
+    private fun <T : IrElement> MutableList<T>.transform(transformation: (T) -> IrElement) {
```

Comment:
Will JVM inliner crash if this function is `inline` with `reified T`?

Reply:
No. It seems information for inline is missed.